nilearn/reporting/html_report.py

```diff
@@ -233,13 +233,25 @@ def generate_report(estimator):
 
 class HTMLReport(HTMLDocument):
     """A report written as HTML.
-    Methods such as save_as_html(), open_in_browser()
-    are inherited from HTMLDocument
+
+    Methods such as ``save_as_html``, or ``open_in_browser``
+    are inherited from :class:`~nilearn.plotting.HTMLDocument`.
 
     """
     def __init__(self, head_tpl, body, head_values={}):
-        """The head_tpl is meant for display as a full page, eg writing on
-        disk. The body is used for embedding in an existing page.
+        """Constructor the ``HTMLReport`` class.
+
+        Parameters
+        ----------
+        head_tpl : Template
```

Comment:
Can the be made more precise ?
Thx ! 
LGTM otherwise.

Reply:
Indeed, this is not super precise... 😅 
I added a bit more infos in [29e66ec](https://github.com/nilearn/nilearn/pull/3030/commits/29e66ece654aab4206ceaf4d71d4dee93818c7ee)